lib/Client.js

```diff
@@ -70,6 +70,38 @@ Client.validate = function (token, cb) {
   })
 }
 
+/**
+ * Validates an access token
+ * @param  {String}   token Token to validate
+ * @param  {String}   client Client Token
+ * @param  {Function} cb    (error)
+ */
+Client.validateClient = function (token, client, cb) {
+  var host = this.host || defaultHost
+  utils.call(host, 'validate', {
+    accessToken: token,
+    clientToken: client
+  }, function (err, data) {
+    cb(err)
+  })
+}
```

Comment:
I there is a significant difference between `token` and `client` which is easily distinguishable then I propose to stay with one `.validate(clientOrAccessToken, cb)` method and detect which one is used.

Reply:
It does need both access and client token though, so you can't really just have a single param

Reply:
I understand, then `validateClient` is okay